packages/aragon-wrapper/src/rpc/handlers/get-forwarded-actions.js

```diff
@@ -0,0 +1,22 @@
+import { map, filter } from 'rxjs/operators'
+
+export default function (request, proxy, wrapper) {
+  // filter out actions that aren't targeted
+  // at the subscribed app proxy
+  const getProxyActions = actionsArray => (
+    actionsArray
+      .filter(action => (action.target === proxy.address))
+  )
+
+  return wrapper.forwardedActions.pipe(
+    // transform the observable into an event-like object
+    // that only contains actions targeting the proxy
+    map(actions => ({
+      event: 'ForwardedActions',
+      returnValues: getProxyActions(actions)
```

Comment:
I wouldn't do this transformation here. If you need it inside the script (not sure if I would recommend it; perhaps this could be a frontend only thing? I still need to think a bit on this), we can add it to `@aragon/api` like we do with accounts.

Reply:
Because we're moving this endpoint into the store, should I leave this here now?